### PR TITLE
Add/card refund details

### LINKED ISSUE #514: componente detalhe do estorno de transação de cartão de crédito
### Exemplo

screenshot from 2018-04-16 18-08-21

### Descrição técnica

```jsx
<CardRefundDetails 
   title=""
   holderName=""
   email=""
   installments=""
   first=""
   last=""
   brand=""
   amount=""
   refundAmount=""
/>
```

### PR DESCRIPTION
**Pull request by rsmelo**
resolves #514

### CODE REVIEW

**Comment by rsmelo** [comment]
@derekstavis  pra esse caso https://app.zeplin.io/project/59e66a0aa633e7a9cf5c0cf4/screen/5ad4f623d3adac460d5e80cf

File: packages/pilot/src/components/CreditCardRefundDetails/index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            </Col>
+
+            <Col palm={12} tablet={6} desk={6} tv={6}>
+              {data.amount}
+            </Col>
+
+            <Col palm={12} tablet={6} desk={6} tv={6}>
+              <Property
+                title={labels.refundAmount}
+                value={<span className={style.refundAmount}>{contents.refundAmount}</span>
```
Comment (lucasjs) [inline packages/pilot/src/components/CreditCardRefundDetails/index.js]: Quebrar linha.

File: packages/pilot/src/components/CreditCardRefundDetails/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            </Col>
+          </Row>
+        </Grid>
+      </CardContent>
+    </Card>
+  )
+}
+
+const shape = {
+  holderName: PropTypes.string.isRequired,
```
Comment (lucasjs) [inline packages/pilot/src/components/CreditCardRefundDetails/index.js]: Ordem alfabética.

File: packages/pilot/src/components/CreditCardRefundDetails/style.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.refundAmount {
+  font-weight: bold;
```
Comment (lucasjs) [inline packages/pilot/src/components/CreditCardRefundDetails/style.css]: Ordem alfabética.

**Pull request merged**